src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -234,6 +287,8 @@
 	public static final RegistryObject<Item> UMBRELLA = REGISTER.register("umbrella", () -> new WeaponItem(new WeaponItem.Builder(Tiers.WOOD, 2, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(InventoryTickEffect.BREATH_SLOW_FALLING), new Item.Properties().defaultDurability(350).tab(MSItemGroup.WEAPONS)));
 	public static final RegistryObject<Item> UPPER_CRUST_CRUST_CANE = REGISTER.register("upper_crust_crust_cane", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.ORGANIC_TIER, 3, -2.0F).efficiency(2.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.SPAWN_BREADCRUMBS).setEating(FinishUseItemEffect.SPAWN_BREADCRUMBS, FinishUseItemEffect.foodEffect(4, 0.5F)), new Item.Properties().tab(MSItemGroup.WEAPONS)));
 	public static final RegistryObject<Item> IRON_CANE = REGISTER.register("iron_cane", () -> new WeaponItem(new WeaponItem.Builder(Tiers.IRON, 2, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL), new Item.Properties().tab(MSItemGroup.WEAPONS)));
+	public static final RegistryObject<Item> KISSY_CUTIE_HEART_HITTER = REGISTER.register("kissy_cutie_heart_hitter", () -> new WeaponItem(new WeaponItem.Builder(Tiers.DIAMOND, 2, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).set(ItemRightClickEffect.switchTo(MSItems.KISSY_CUTIE_HEART_SPLITTER)), new Item.Properties().tab(MSItemGroup.WEAPONS).rarity(Rarity.UNCOMMON)));
+	public static final RegistryObject<Item> KISSY_CUTIE_HEART_SPLITTER = REGISTER.register("kissy_cutie_heart_splitter", () -> new WeaponItem(new WeaponItem.Builder(Tiers.DIAMOND, 4, -2.6F).efficiency(10.0F).disableShield().set(MSItemTypes.SICKLE_TOOL).set(ItemRightClickEffect.switchTo(MSItems.KISSY_CUTIE_HEART_HITTER)), new Item.Properties().tab(MSItemGroup.WEAPONS).rarity(Rarity.UNCOMMON)));
```

Comment:
Seems a little awkward with a scythe sitting in the middle of the canes category. How do you feel about moving it to the scythes, despite its connection to the Kissy Cutie Heart Hitter?

Reply:
fuck i didnt mean t0 res0lve that last one!! 00ps. in regards t0 that last 0ne, the hellbringers h0e, its fr0m the mindset that all h0es will be scythes, but n0t all scythes will be h0es! 
the kissy cutie heartsplitter can be m0ved t0 the scythes, yes :)